test/pkg/testconfig/testconfig.go

```
@@ -868,21 +935,32 @@ func CreatePtpConfigGrandMaster(nodeName, ifName string) error {
+}
+
+func CreatePtpConfigWPCGrandMaster(policyName string, nodeName string, ifList []string, deviceID string, label string) error {
 	ptpSchedulingPolicy := SCHED_OTHER
 	configureFifo, err := strconv.ParseBool(os.Getenv("CONFIGURE_FIFO"))
 	if err == nil && configureFifo {
 		ptpSchedulingPolicy = SCHED_FIFO
 	}
 	// Sleep for a second to allow previous label on the same node to complete
 	time.Sleep(time.Second)
-	_, err = nodes.LabelNode(nodeName, pkg.PtpClockUnderTestNodeLabel, "")
+	_, err = nodes.LabelNode(nodeName, label, "")
```

golangci-lint (2.12.2)

[error] ineffectual assignment to err

(ineffassign)